### PR TITLE
Script Wiki Updates

### PR DESCRIPTION
**Pull request by Herobrine643928**
Updates to the modules used within the script API section of the wiki
- Version updates from 1.19.80 to 1.21.20.
- Module version updates from 1.4.0-beta to 1.13.0 wherever possible, and 1.14.0-beta for chat events mainly.
- Use of stable script features wherever possible to allow for less edits later on.
- Minor grammar & formatting fixes over all pages
- Change to the ordering of the main script pages on the sidebar.

### CODE REVIEW

**Review by ThomasOrs — APPROVED** [review]
LGTM! :shipit:

**Comment by JaylyDev** [comment]
I'd add BigInt into the engine support list.

File: docs/scripting/script-net.md [hunk 1]
```diff
@@ -11,14 +11,14 @@ mentions:
 ---
 
 ::: warning
-The Script API is currently in active development, and breaking changes are frequent. This page assumes the format of Minecraft 1.19.80
+The Script API is currently in active development, and breaking changes are frequent. This page assumes the format of Minecraft 1.21.20
 :::
 
 ::: warning
-This module can only be used on Bedrock Dedicated Server.
+This module can only be used on Bedrock Dedicated Servers.
```
Comment (QuazChick) [inline docs/scripting/script-net.md:18]: Native module `@minecraft/server-net` can also be used in Editor extensions.

Comment (SuperLlama88888) [reply docs/scripting/script-net.md:18]: @QuazChick Hypothetically, could you make an Editor addon that uploads a file to a server?

**Pull request merged**